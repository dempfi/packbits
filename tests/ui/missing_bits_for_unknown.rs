use packbits as _;

#[packbits::pack(bytes = 1)]
struct Bad {
    a: u64,
}

fn main() {}
