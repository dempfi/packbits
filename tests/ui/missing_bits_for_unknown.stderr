error: missing #[bits(W)] for this field type
 --> tests/ui/missing_bits_for_unknown.rs:5:5
  |
5 |     a: u64,
  |     ^
